src/command_modules/azure-cli-sf/azure/cli/command_modules/sf/commands.py

```diff
@@ -13,12 +13,39 @@
 cluster_operations = create_service_adapter("azure.servicefabric",
                                             "ServiceFabricClientAPIs")
 
-# No client for specific custom commands
+# Custom commands
+# TODO: Fix custom commands to accept client as additional argument instead of
+# generating
 cli_command(__name__, "sf cluster select",
             "azure.cli.command_modules.sf.custom#sf_select")
 cli_command(__name__, "sf application upload",
             "azure.cli.command_modules.sf.custom#sf_upload_app")
+cli_command(__name__, "sf compose create",
+            "azure.cli.command_modules.sf.custom#sf_create_compose_application")
+cli_command(__name__, "sf service create",
+            "azure.cli.command_modules.sf.custom#sf_create_service")
+cli_command(__name__, "sf service update",
+            "azure.cli.command_modules.sf.custom#sf_update_service")
+cli_command(__name__, "sf service report-health",
+            "azure.cli.command_modules.sf.custom#sf_report_svc_health")
+cli_command(__name__, "sf application create",
+            "azure.cli.command_modules.sf.custom#sf_create_app")
+cli_command(__name__, "sf application report-health",
+            "azure.cli.command_modules.sf.custom#sf_report_app_health")
+cli_command(__name__, "sf application upgrade",
+            "azure.cli.command_modules.sf.custom#sf_upgrade_app")
+cli_command(__name__, "sf node report-health",
+            "azure.cli.command_modules.sf.custom#sf_report_node_health")
+cli_command(__name__, "sf node service-package-upload",
+            "azure.cli.command_modules.sf.custom#sf_service_package_upload")
+cli_command(__name__, "sf replica report-health",
+            "azure.cli.command_modules.sf.custom#sf_report_replica_health")
+cli_command(__name__, "sf chaos start",
+            "azure.cli.command_modules.sf.custom#sf_start_chaos")
+cli_command(__name__, "sf partition report-health",
+            "azure.cli.command_modules.sf.custom#sf_report_partition_health")
```

Comment:
Why do you move them out? For registering custom command? It can be done with the context module as well:
https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py#L24